src/test/java/com/epam/esm/service/GiftCertificateServiceTest.java

```diff
@@ -77,11 +83,11 @@ public void testShouldReturnListOfGiftCertificates() throws EntityNotExistsExcep
         //given
         List<GiftCertificate> expectedGiftCertificates = Arrays.asList(expectedGiftCertificate, expectedGiftCertificate);
         List<GiftCertificateDto> expectedGiftCertificateDtos = Arrays.asList(expected, expected);
-        when(giftCertificateDao.getGiftCertificates(null, null, null, null, null)).thenReturn(expectedGiftCertificates);
+        when(giftCertificateDao.getGiftCertificates(null, null, null, null, null, null, null)).thenReturn(expectedGiftCertificates);
         when(dtoMapper.map(expectedGiftCertificate)).thenReturn(expected);
 
         //when
-        List<GiftCertificateDto> actual = service.getGiftCertificates(null, null, null, null, null);
+        List<GiftCertificateDto> actual = service.getGiftCertificates(null, null, null, null, null, null, null);
```

Comment:
`service.getGiftCertificates` - method's signature became way too complex. We need to fix this. Try to wrap som parameters

Reply:
used less parameters